src/constants.rs

```diff
@@ -88,3 +88,180 @@ pub const NFULA_CT_INFO: u16 = libc::NFULA_CT_INFO as u16;
 
 pub const NFULNL_MSG_CONFIG: u8 = libc::NFULNL_MSG_CONFIG as u8;
 pub const NFULNL_MSG_PACKET: u8 = libc::NFULNL_MSG_PACKET as u8;
+
+// netflter/nfnetlink_conntrack.h
+// There is no definitions in rust-lang/libc
+pub const IPCTNL_MSG_CT_NEW: u8 = 0;
```

Comment:
The netlink-packet-route has stopped exposing constants out.

I do not have time to polish here yet, but please do not add more lines to `src/contants.rs`.
Please:
 1. Remoev `pub`.
 2. Move constant to its user, this make our review easier.